boot/debug.go

```diff
@@ -83,3 +87,44 @@ func DumpBootVars(w io.Writer, dir string, uc20 bool) error {
 	}
 	return nil
 }
+
+// DebugSetBootVars is a debug helper that takes a list of <var>=<value> entries
+// and sets them for the configured bootloader.
+func DebugSetBootVars(dir string, recoveryBootloader bool, varEqVal []string) error {
+	opts := &bootloader.Options{
+		NoSlashBoot: dir != "" && dir != "/",
```

Comment:
maybe just to make this absolutely clear to the reader you could also add `Role: bootloader.RoleSole` here so it's clear that if none of the uc20 paths are taken, the default value will be RoleSole and will make sense

```suggestion
		NoSlashBoot: dir != "" && dir != "/",
		Role:        bootloader.RoleSole
```